services/nublado/values-idfdev.yaml

```diff
@@ -0,0 +1,103 @@
+
+  oauth_provider: 'jwt'
+
+  hub:
+    image: 'lsstsqre/sciplat-hub:latest'
+
+  lab:
+    restrict_nodes: 'true'
+    image:
+      experimentals: 2
+      dailies: 2
+      cachefile: '/home/jovyan/repo-cache.json'
+      repo_username: 'sqreadmin'
+
+    resources:
+      nodejs_max_mem: '6144'
+      mb_per_cpu: 3072
+
+  routes:
+    soda: '/api/image/soda'
+    external:
+      instance: 'https://science-platform-dev.lsst.cloud'
+
+  dask:
+    restrict_nodes: 'true'
+
+  vault_secrets:
+    enabled: true
+    path: secret/k8s_operator/science-platform-dev.lsst.cloud/nublado
+
+  wf:
+    ingress_ui:
+      annotations:
+        nginx.ingress.kubernetes.io/auth-method: GET
+        nginx.ingress.kubernetes.io/auth-response-headers: X-Auth-Request-Token
+        nginx.ingress.kubernetes.io/auth-signin: "https://science-platform-dev.lsst.cloud/login"
+        nginx.ingress.kubernetes.io/auth-url: "https://science-platform-dev.lsst.cloud/auth?scope=exec:notebook"
+        nginx.ingress.kubernetes.io/configuration-snippet: |
+          proxy_set_header X-Forwarded-Proto https;
+          proxy_set_header X-Forwarded-Port 443;
+          proxy_set_header X-Forwarded-Path /workflow;
+          auth_request_set $auth_token $upstream_http_x_auth_request_token;
+          proxy_set_header X-Portal-Authorization "Bearer $auth_token";
+          error_page 403 = "/auth/forbidden?scope=exec:notebook";
+
+  prepuller:
+    schedule: "27 01,07,13,19 * * *"
+
+  proxy:
+    ingress:
+      annotations:
+        nginx.ingress.kubernetes.io/auth-method: GET
+        nginx.ingress.kubernetes.io/auth-response-headers: X-Auth-Request-Token
+        nginx.ingress.kubernetes.io/auth-signin: "https://science-platform-dev.lsst.cloud/login"
+        nginx.ingress.kubernetes.io/auth-url: "https://science-platform-dev.lsst.cloud/auth?scope=exec:notebook"
+        nginx.ingress.kubernetes.io/configuration-snippet: |
+          proxy_set_header X-Forwarded-Proto https;
+          proxy_set_header X-Forwarded-Port 443;
+          proxy_set_header X-Forwarded-Path /nb;
+          auth_request_set $auth_token $upstream_http_x_auth_request_token;
+          proxy_set_header X-Portal-Authorization "Bearer $auth_token";
+          error_page 403 = "/auth/forbidden?scope=exec:notebook";
+      host: science-platform-dev.lsst.cloud
+      tls:
+        cluster_issuer: cert-issuer-letsencrypt-dns
+        secret: nublado-tls-secret
+        tls_host: science-platform-dev.lsst.cloud
+
+  mountpoints: |
+    [
+      { "disabled": true,
+        "mountpoint": "/example -- if it doesn't start with '/' we add it",
+        "fileserver-host": "omit to use Hub settings",
+        "fileserver-export": "defaults to '/exports<mountpoint>'",
+        "mode": "rw or ro, defaults to 'ro'",
+        "options": "mount options, e.g. 'local_lock=all'"
+      },
+      {
+       "mountpoint": "/home",
+        "mode": "rw",
+        "fileserver-host": "FIXME",
```

Comment:
Do you have a value for the fileserver-host?

Reply:
I think you're looking at only the first commit.

10.210.11.194